website/docs/d/ecs_service.html.markdown

```diff
@@ -0,0 +1,65 @@
+---
+layout: "aws"
+page_title: "AWS: aws_ecs_service"
+sidebar_current: "docs-aws-datasource-ecs-service"
+description: |-
+    Provides details about an ecs service
+---
+
+# Data Source: aws_ecs_service
+
+The ECS Service data source allows access to details of a specific
+Service within a AWS ECS Cluster.
+
+## Example Usage
+
+```hcl
+data "aws_ecs_service" "ecs-mongo" {
+  service_name = "${aws_ecs_service.mongo.name}"
+  cluster_arn  = "${aws_ecs_cluster.foo.arn}"
+}
+
+resource "aws_ecs_cluster" "foo" {
```

Comment:
Normally we try to not show an unexpected configuration for newer operators (e.g. the data source directly referencing its resource counterpart) -- in this case its fine to simply just list the example as:

```hcl
data "aws_ecs_service" "example" {
   cluster_arn  = "${data.aws_ecs_cluster.example.arn}"
   service_name = "example"
 }
```